.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @open-turo/github-actions
```

Comment:
You can change this to:
```suggestion
* @open-turo/it-sec
```

Reply:
Addressed [3bd5edf](https://github.com/open-turo/actions-security/pull/1/commits/3bd5edfdf2bdf4275fe9b40dedd6514e8d11920a)